src/authentication/renderer/AuthenticationApp.vue

```diff
@@ -241,4 +252,9 @@ async function login() {
 .submit-button {
 	margin-top: 0.5rem;
 }
+
+:deep(.login-box__server--predefined) {
```

Comment:
styles vary a bit, but that might be upstream issue
loading | error
-- | --
![image](https://github.com/user-attachments/assets/4718ac7e-0b1d-4049-9539-de6aa7889051) | ![image](https://github.com/user-attachments/assets/b503adcf-defe-4323-90b4-873736984d93)

Reply:
I didn't understand. Do you mean the font color?